### ISSUE TITLE
bug(ripple): TypeError: Cannot read property 'unbounded' of null 

### ISSUE THREAD

**Issue by mgr34** [jamesmfriedman/rmwc#565]
I was upgrading to 6.0.0-alpha.13 (from alpha.12) this morning and ran into the above error message. Thinking it was just a larger bug in my project I checked out the project and tried to load the documentation and was met with the same error. Still investigating the source. Will update my issue if/when I find more information.

```bash
git clone https://github.com/jamesmfriedman/rmwc.git
cd rmwc/
nvm use 12.16.1
git fetch --all --tags --prune
git checkout tags/v6.0.0-alpha.13
yarn && yarn start
```

![2020-03-16-102445_1023x452_scrot](https://user-images.githubusercontent.com/9339171/76768133-f86b4880-6770-11ea-8cc3-1a8f326d7ddc.png)

edit 1: In some cases getProps() returns null. Still groking this library.

**Comment by mgr34** [comment]
If I am understanding this correctly this occurs when `useEffect` resets props.current to null. 

https://github.com/jamesmfriedman/rmwc/blob/158b04689970c533e176c43825a96c99e0770aeb/src/base/foundation-component.tsx#L298

At least changing to 
`props.current = {}`

appears to resolve the issue on my end. 

(Still investigating)

**Comment by jamesmfriedman** [comment]
Found this exact bug this morning, and the exact same fix. Publishing a new alpha shortly

**Comment by mgr34** [comment]
Great Thanks! I'll close this issue.

**Comment by jamesmfriedman** [comment]
@mgr34 published 6.0.0-alpha.14

**Comment by mgr34** [comment]
@jamesmfriedman I see that Thanks. 

FYI -- I just asked a question in discord about submitting a PR for another issue.
